crates/replace/src/lib.rs

```diff
@@ -349,13 +336,10 @@ impl<T: Config> Pallet<T> {
         );
 
         // check that that the total griefing collateral is sufficient
-        let required_collateral = ext::fee::get_replace_griefing_collateral::<T>(
+        let griefing_collateral = ext::fee::get_replace_griefing_collateral::<T>(
```

Comment:
```suggestion
        let griefing_collateral = ext::fee::get_replace_griefing_collateral::<T>(to_be_replaced_increase);
```
And remove `let replace_collateral_increase = griefing_collateral.saturating_sub(&previous_griefing_collateral)?;`